config/initializers/okcomputer.rb

```diff
@@ -0,0 +1,22 @@
+require 'honeybadger'
+require 'net/smtp'
+
+class SmtpCheck < OkComputer::Check
+  def check
+    smtp = Net::SMTP.new ENV['ACTION_MAILER_SMTP_ADDRESS'], ENV['ACTION_MAILER_PORT']
+    smtp.enable_starttls
+    smtp.start('curationexperts.com', ENV['ACTION_MAILER_USER_NAME'], ENV['ACTION_MAILER_PASSWORD'], :plain) do
+      mark_message "SMTP connection working"
+    end
+  rescue => exception
+    Honeybadger.notify(exception)
+    mark_failure
+    mark_message "Cannot connect to SMTP"
+  end
+end
+
+# OkComputer::Registry.register "action_mailer", OkComputer::ActionMailer.new
+OkComputer::Registry.register "batch_sidekiq_queue_latency", OkComputer::SidekiqLatencyCheck.new('batch', 360)
```

Comment:
Is there a reason you're not checking the ingest queue here as well?

https://github.com/curationexperts/epigaea/blob/master/config/sidekiq.yml#L3-L6